src/bundle/Resources/public/scss/_content-edit.scss

```diff
@@ -5,40 +5,128 @@
     position: relative;
+    width: 100%;
+    padding-top: calculateRem(85px);
 
     &__close {
-        position: absolute;
-        left: 0;
-        top: 0;
-        font-size: calculateRem(32px);
+        position: fixed;
+        z-index: 3;
+        top: calculateRem(6px);
+        left: calculateRem(16px);
+        width: calculateRem(40px);
+        height: calculateRem(40px);
         color: $ez-black;
-        width: calculateRem(32px);
-        height: calculateRem(32px);
-        margin-left: calculateRem(8px);
-        padding: calculateRem(16px);
-
-        &:before,
-        &:after {
-            content: '';
-            width: calculateRem(32px);
-            height: calculateRem(2px);
-            position: absolute;
-            background-color: $ez-black;
-            top: 50%;
-            left: 0;
-        }
+        line-height: calculateRem(40px);
+    }
 
-        &:before {
-            transform: rotate(45deg);
-        }
+    .ez-edit-header {
+        position: fixed;
+        z-index: 2;
+        top: 0;
+        left: 0;
+        width: calc(100% - #{calculateRem(80px)});
 
-        &:after {
-            transform: rotate(-45deg);
+        &__content-type-name {
+            min-height: calculateRem(55px);
+            padding: calculateRem(12px);
+            color: $ez-black;
+            font-size: calculateRem(20px);
+            font-weight: bold;
+            background: $ez-color-base-pale;
         }
 
-        &:hover {
-            text-decoration: none;
+        &__details {
             color: $ez-black;
+            font-size: calculateRem(12px);
+            background: $ez-ground-base-medium;
+            transition: max-height 0.3s;
+            overflow: hidden;
+
+            .ez-details-items {
+                padding: 0 calculateRem(48px);
+                max-height: calculateRem(32px);
+                transition: max-height 0.3s;
+
+                &__pill {
+                    display: inline-block;
+                    margin: calculateRem(2px) calculateRem(3px);
+                    padding: calculateRem(1px) calculateRem(2px);
+
+                    font-weight: 100;
+
+                    -webkit-border-radius: calculateRem(4px);
+                    -moz-border-radius: calculateRem(4px);
+                    border-radius: calculateRem(4px);
+
+                    &--gray {
+                        padding: calculateRem(4px) calculateRem(4px);
+                        color: $ez-color-base-medium;
+                        background: $ez-white;
+
+                        .ez-icon {
+                            fill: $ez-color-base-medium;
+                        }
+                    }
+
+                    &--white {
+                        color: $ez-white;
+                        font-style: italic;
+                        background: $ez-color-base-light;
+                    }
+
+                    &--teal {
+                        color: $ez-ground-primary;
```

Comment:
Based on the screenshots you have attached (thank you!) it gives the impression that it's disabled. We need more color contrast here. I'd suggest to set the color to `$ez-secondary-ground-dark`